.ci/jobs/apm-agent-ruby-mbp.yml

```diff
@@ -5,8 +5,8 @@
     description: APM Agent Ruby
     script-path: .ci/Jenkinsfile
     scm:
-    - github:
-        branch-discovery: all
+    - githu:
```

Comment:
typo?

Reply:
sorry, I didn't put a comment in the change itself but in the commit, as I do want to validate the pipeline changes as I just added a `sanity checks` stage to evaluate and detect those errors within the pipeline

Reply:
Note: this is only for testing purposes